lib/world_cup.rb

```diff
@@ -0,0 +1,76 @@
+require "pry"
+require "./lib/team"
+
+class WorldCup
+  attr_reader :year, :teams
+  def initialize(year, array_of_teams)
+    @year = year
+    @teams = array_of_teams
+  end
+
+  def active_teams
+    @teams.select do |team|
+      team.eliminated == false
+    end
+  end
+
+  def active_players_by_position(position_on_field)
+    return_array = []
+    active_teams.select do |team|
+      team.players.select do |player|
```

Comment:
You already wrote a method for a team to give you all it's players by position. Just call that method here.